doc/feature/httpServer.md

```diff
@@ -0,0 +1,144 @@
+HTTP Server for ZeroEq Events
+============
+
+The http server implements a zeq::Receiver and Sender to serve ZeroBuf
```

Comment:
The idea is nice but before implementing anything, I highly encourage "us" to do some research what is there already.
- The http server implementation is scary ( although it can be 30 lines: http://hintjens.com/blog:42 ). When, for instance https etc is needed we may need to look for other solutions.
- There are some libraries like http://mongrel2.org/ and/or https://github.com/fanout/zurl we may need to look as well.
- For the JSON ( as we have decided text over cable protocol ), we may check possible JSON libraries ( mongrel2 has it I think )